src/tokenizer/tokenizer_manager.rs

```diff
@@ -64,12 +65,14 @@ impl Default for TokenizerManager {
         manager.register(
             "default",
             SimpleTokenizer
+                .filter(StopWordFilter::default())
                 .filter(RemoveLongFilter::limit(40))
                 .filter(LowerCaser),
         );
         manager.register(
             "en_stem",
             SimpleTokenizer
+                .filter(StopWordFilter::default())
```

Comment:
I don't think applying stop words by default is a good idea.

Reply:
Coming from Lucene I would expect to see a default list of stop words removed when indexing content. The words in this list are pretty low value. Do you have a use case in mind that I'm not considering? Either way, I can remove it - it would be pretty quick.

Reply:
Then that's an error in lucene... How do you search for the band "the who" in such an index?

Reply:
You can add another analyzer called en_stem_nostopwords or something like that maybe?

Reply:
I think we can totally wait for our user base to let us know. ;p But yes, those types of phrases are tricky when using stopwords. :)